Cmdline/Action/Install.cs

```diff
@@ -129,7 +129,7 @@ public int RunCommand(CKAN.KSP ksp, object raw_options)
 
                 string message = String.Format("Too many mods provide {0}. Please pick from the following:\n", ex.requested);
 
-                int result = -1;
+                int result;
```

Comment:
Just to check, wouldn't this change the code below in the case where:

- Our `RaiseSelectionDialog` throws an exception, and
- That exception is not a Kraken?

In those cases we're checking if `result < 0` down at line 145.